### ISSUE TITLE
`css.format.enable` -> `css.format.enabled`

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#145763]
Testing #145604

elsewhere we use `enabled`, so think we should also do that here

This also makes sense because `enable` is an action and `enabled` is a state

**Comment by aeschli** [comment]
Unfortunately "xy.enable" and "yz.enabled" are both used widely in our settings (easy to very in the Default Settings JSON page).

I chose `enable` to stay consistent with `javascript.format.enable`, `hml.format.enable` and `json.format.enable`

**Comment by meganrogge** [comment]
Do you know if there's an issue to align those?

**Comment by aeschli** [comment]
No, I don't know. I personally think that it would be quite a swirl and not worth the effort.
